### PR TITLE
fix: family details date of birth issue

### PR DESCRIPTION
**Pull request by PC-11-00**
## Description
now we can select the dob of a family member before 2000 year also

## Related issues and discussion
#1682 

## Screenshots, if any

https://user-images.githubusercontent.com/76156941/218531968-0e405384-0d1f-4fcb-a39f-5ad1a397c1a2.mov

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by Subzer0570** [comment]
Hello @PC-11-00, myself Udit Damare 
I am not getting my both FE(webapp) and BE(fineract) server running on both local development
For webapp
<--- JS stacktrace --->
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

And for fineract it shows bunch of errors and first error is MYSQL error  and ending with with unable to start embedded tomcat server.
I'll share gist of it after this msg.
I have installed mysql 5.6 and node version is 18. (something)
I havent made any changes in any repo
Please help me get started. I have wasted so much time on this

**Comment by Subzer0570** [comment]
https://gist.github.com/Subzer0570/90473c01ec1afd62b1ffd6f91d7f67fd

**Comment by PC-11-00** [comment]
@Subzer0570 come on slack and don't comment randomly on any PR

**Comment by Subzer0570** [comment]
I did post it first on slack, but people didnt seem to respond there, thats why ai tried over here

**Comment by Subzer0570** [comment]
I did post it first on slack, but people didnt seem to respond there, thats why ai tried over here

**Comment by bharathcgowda** [comment]
@adamsaghy or @josehernandezfintecheandomx  could any one of you help in reviewing this PR code changes , please?

File: src/app/clients/client-stepper/client-family-members-step/client-family-member-dialog/client-family-member-dialog.component.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ import { Dates } from 'app/core/utils/dates';
 export class ClientFamilyMemberDialogComponent implements OnInit {
 
   /** Minimum Due Date allowed. */
-  minDate = new Date(2000, 0, 1);
+  // minDate = new Date(2000, 0, 1);
```
Comment (adamsaghy) [inline src/app/clients/client-stepper/client-family-members-step/client-family-member-dialog/client-family-member-dialog.component.ts:21]: Feel free to remove it.

File: src/app/clients/clients-view/family-members-tab/add-family-member/add-family-member.component.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ import { Dates } from 'app/core/utils/dates';
 export class AddFamilyMemberComponent implements OnInit {
 
   /** Minimum Due Date allowed. */
-  minDate = new Date(2000, 0, 1);
+  // minDate = new Date(2000, 0, 1);
```
Comment (adamsaghy) [inline src/app/clients/clients-view/family-members-tab/add-family-member/add-family-member.component.ts:22]: Feel free to remove it.

File: src/app/clients/clients-view/family-members-tab/edit-family-member/edit-family-member.component.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ import { Dates } from 'app/core/utils/dates';
 export class EditFamilyMemberComponent implements OnInit {
 
   /** Minimum Due Date allowed. */
-  minDate = new Date(2000, 0, 1);
+  // minDate = new Date(2000, 0, 1);
```
Comment (adamsaghy) [inline src/app/clients/clients-view/family-members-tab/edit-family-member/edit-family-member.component.ts:22]: Feel free to remove it.

File: src/app/clients/client-stepper/client-family-members-step/client-family-member-dialog/client-family-member-dialog.component.ts (hunk 1 above)
Comment (PC-11-00) [reply src/app/clients/client-stepper/client-family-members-step/client-family-member-dialog/client-family-member-dialog.component.ts:21]: okay

File: src/app/clients/clients-view/family-members-tab/add-family-member/add-family-member.component.ts (hunk 2 above)
Comment (PC-11-00) [reply src/app/clients/clients-view/family-members-tab/add-family-member/add-family-member.component.ts:22]: okay

File: src/app/clients/clients-view/family-members-tab/edit-family-member/edit-family-member.component.ts (hunk 3 above)
Comment (PC-11-00) [reply src/app/clients/clients-view/family-members-tab/edit-family-member/edit-family-member.component.ts:22]: okay

**Review by adamsaghy — APPROVED** [review]
LGTM

**Pull request merged**